elm.json

```diff
@@ -3,15 +3,15 @@
     "name": "miniBill/elm-generic-dict",
     "summary": "Codegen dictionaries with arbitrary key types.",
     "license": "BSD-3-Clause",
-    "version": "1.0.1",
+    "version": "1.0.2",
     "exposed-modules": [
         "GenericDict"
     ],
     "elm-version": "0.19.0 <= v < 0.20.0",
     "dependencies": {
         "elm/core": "1.0.2 <= v < 2.0.0",
         "elm-community/string-extra": "4.0.1 <= v < 5.0.0",
-        "mdgriffith/elm-codegen": "2.0.0 <= v < 3.0.0"
+        "mdgriffith/elm-codegen": "2.0.0 <= v < 4.0.1"
```

Comment:
I never knew this was allowed (is it?)

Reply:
There very well could be, but I'm not sure! I deleted my `elm-stuff` folder and rebuilt everything and it didn't seem to complain, but there's a lot I don't know.

Reply:
I think it's actually a bad idea to do that unless you explicitly check that it does work with all the major versions. I've just bumped it to 4.0.0